packages/integration-karma/test/polyfills/text-content/index.spec.js

```diff
@@ -0,0 +1,16 @@
+describe('Node.textContent', () => {
+    it('should not return comment text when Node.nodeType is ELEMENT_NODE', () => {
```

Comment:
add another test that has the context in the second level, not as a root element, just to make sure that we cover that too.